docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -46,9 +46,9 @@ yarn add @material-ui/core@next
 ### Update React version
 
 The minimum required version of React was increased from `react@^16.3.0` to `react@^16.8.0`.
-This allows us to rely on [Hooks](https://reactjs.org/docs/hooks-intro.html).
+This allows us to rely on [Hooks](https://reactjs.org/docs/hooks-intro.html) (we no longer use the class API).
```

Comment:
It might be a bit too early to claim this :P

Reply:
@joshwooding It will eventually be true thanks to you 😍.